src/config/effect.rs

```diff
@@ -27,20 +36,55 @@ pub struct Delay {
 }
 
 impl Delay {
-    fn to_audio_unit(&self, stream_config: &StreamConfig) -> Result<audio_unit::Boxed> {
+    fn to_audio_unit(
+        &self,
+        midi_config: &Option<config::Midi>,
+        stream_config: &StreamConfig,
+    ) -> Result<audio_unit::Boxed> {
+        let num = self.num;
+        let channel = match midi_config {
+            Some(midi_config) => midi_config.channel()?,
+            None => Channel::Ch1,
```

Comment:
This seems like a clunky place for a default MIDI channel # to live (assuming that it would also apply to other effects)?

Reply:
👍🏻 , moved that into the midi config.